tests/retryable-reads/retryable-reads-001.phpt

```diff
@@ -26,14 +26,15 @@ class Observer implements MongoDB\Driver\Monitoring\CommandSubscriber
 }
 
 $manager = new MongoDB\Driver\Manager(URI, ['retryReads' => true]);
+$server = $manager->selectServer(new \MongoDB\Driver\ReadPreference('primary'));
 
 $bulk = new MongoDB\Driver\BulkWrite;
 $bulk->insert(['x' => 1]);
 $bulk->insert(['x' => 2]);
 
-$manager->executeBulkWrite(NS, $bulk);
+$server->executeBulkWrite(NS, $bulk);
 
-configureFailPoint($manager, 'failCommand', ['times' => 1], ['failCommands' => ['aggregate'], 'closeConnection' => true]);
+configureTargetedFailPoint($server, 'failCommand', ['times' => 1], ['failCommands' => ['aggregate'], 'closeConnection' => true]);
```

Comment:
For my own curiosity, should this test originally have had `skip_if_multiple_mongos()`, since it set a fail point and didn't also require a replica set?

Reply:
Correct. Since our sharded clusters only had a single mongos, there was no problem with the fail point. Now that there are more of them, we have to create it on a specific server.